routing/routing.xml

```diff
@@ -725,76 +725,85 @@
 		<way attribute="obstacle_srtm_alt_speed">
 			<if param="height_obstacles">
 				<gt value1="-100" value2=":incline">
+					<select value="-1" t="surface" v="cobblestone"/>
 					<select value="-1" t="osmand_highway_integrity_brouting_low" v="yes"/>
 					<select value="-1" t="highway" v="path"/>
 					<select value="-1" t="highway" v="track"/>
+					<select value="-1" t="highway" v="footway"/>
 				</gt>
 				<gt value1="-25" value2=":incline">
+					<select value="170" t="surface" v="cobblestone"/>
+					<select value="1" t="surface" v="paved"/>
```

Comment:
Yes, tracks and paths are sometimes marked with “asphalt”, "paved" or "concrete". I wanted to make sure that the penalty for bad downhill tracks is not applied to tracks / paths that are clearly marked as paved.